manta-crypto/src/merkle_tree/inner_tree.rs

```diff
@@ -20,11 +20,15 @@
 //       some of it, to reduce potential duplication?
 // TODO: We should probably move `InnerNode` and its related `struct`s to `merkle_tree::node`.
 
-use crate::merkle_tree::{
-    path::{CurrentInnerPath, InnerPath},
-    path_length, Configuration, InnerDigest, Node, Parameters, Parity,
+use crate::{
+    eclair::bool::Bool,
+    merkle_tree::{
+        path::{CurrentInnerPath, InnerPath},
+        path_length, Configuration, InnerDigest, Node, Parameters, Parity,
+    },
 };
-use alloc::collections::btree_map;
+
+use alloc::{collections::btree_map, vec::Vec};
 use core::{fmt::Debug, hash::Hash, iter::FusedIterator, marker::PhantomData, ops::Index};
```

Comment:
Keep the imports together.

Reply:
What do you mean exactly?

Reply:
I mean that there shouldn't be a stray line here (the rustfmt will also reformat the imports as necessary).

https://github.com/Manta-Network/manta-rs/blob/0cb4684778ca17e01d923b69972fabb0dd0caf25/manta-crypto/src/merkle_tree/inner_tree.rs#L32